core/src/utils/test/playwright/generator.ts

```diff
@@ -1,6 +1,13 @@
 export type Mode = 'ios' | 'md';
 export type Direction = 'ltr' | 'rtl';
-export type Theme = 'light' | 'dark';
+/**
+ * The theme to use for the playwright test.
+ *
+ * - `default`: The fallback theme values.
+ * - `light`: The light theme values.
+ * - `dark`: The dark theme values.
+ */
+export type Theme = 'default' | 'light' | 'dark';
```

Comment:
Bikeshedding: I wonder if "none" or "fallback" would make more sense here? I think of light mode as the default theme. Not a blocker for me, so if you think "default" is still the best option I'm fine with that.

Reply:
I'm not wedded to the naming here, also agree `default` isn't clear. I think `none` could make sense 👍